### DISCUSSION TITLE
outreachy Applicant

### DISCUSSION THREAD

**Discussion by angelnagaba** [ModECI/MDF#211] [Q&A] [answered]
Hello
My name is Angel Nagaba. I am finalizing my Bachelor of Science in Software Engineering from Makerere University Kampala Uganda. I am an outreachy applicant and would like to contribute to this project. 

I am excited about this opportunity.

**Comment by sanjayankur31** [comment, marked as answer]
Welcome @angelnagaba ! Please take a look at #213 for initial tasks :rocket:
